src/main/java/com/rtb/projectmanagementtool/project/ProjectController.java

```diff
@@ -0,0 +1,101 @@
+/**
+ * ProjectController.java - this file implements ProjectController, which controls the logic of
+ * working with Project objects.
+ */
+package com.rtb.projectmanagementtool.project;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import java.util.HashSet;
+
+public class ProjectController {
+  private DatastoreService datastore;
+  private HashSet<ProjectData> projects;
+
+  /** Class constructor */
+  public ProjectController() {
+    datastore = DatastoreServiceFactory.getDatastoreService();
+    projects = new HashSet<ProjectData>();
+    loadProjects();
+  }
+
+  /** Initializes the projects HashSet by retrieving all datastore entities of type "Project" */
+  private void loadProjects() {
+    PreparedQuery results = datastore.prepare(new Query("Project"));
```

Comment:
Do we always have to load all the projects every time?  Maybe also think about cases where you'd only need a subset.